libs/journeys/ui/src/components/Card/ContainedCover/ContainedCover.spec.tsx

```diff
@@ -105,7 +102,10 @@ describe('ContainedCover', () => {
     const imageCover = getByTestId('background-image')
 
     expect(imageCover).toHaveAccessibleName(imageBlock.alt)
-    expect(imageCover).toHaveAttribute('src', minifiedUnsplashImage)
+    expect(imageCover).toHaveAttribute(
+      'src',
+      expect.stringContaining(encodeURIComponent(imageBlock.src ?? ''))
```

Comment:
Tests assertions shouldn't be ambiguous 
```suggestion
      expect.stringContaining(encodeURIComponent(imageBlock.src))
```